include/cantera/thermo/MixtureFugacityTP.h

```diff
@@ -1,577 +1,572 @@
+     *     that there is no guessed density
+     * @return    We return the density of the fluid at the requested phase. If
+     *            we have not found any acceptable density we return a -1. If we
+     *            have found an acceptable density at a different phase, we
+     *            return a -2.
+     */
+    virtual doublereal densityCalc(doublereal TKelvin, doublereal pressure, int phaseRequested,
+                                   doublereal rhoguess);
+
+protected:
+    //! Utility routine in the calculation of the saturation pressure
+    /*!
+     * @param TKelvin        temperature (kelvin)
+     * @param pres           pressure (Pascal)
+     * @param[out] densLiq   density of liquid
+     * @param[out] densGas   density of gas
+     * @param[out] liqGRT    deltaG/RT of liquid
+     * @param[out] gasGRT    deltaG/RT of gas
+     */
+    int corr0(doublereal TKelvin, doublereal pres, doublereal& densLiq,
+              doublereal& densGas, doublereal& liqGRT, doublereal& gasGRT);
+
+public:
+    //! Returns the Phase State flag for the current state of the object
+    /*!
+     * @param checkState If true, this function does a complete check to see
+     *        where in parameters space we are
+     *
+     *  There are three values:
+     *  - WATER_GAS   below the critical temperature but below the critical density
+     *  - WATER_LIQUID  below the critical temperature but above the critical density
+     *  - WATER_SUPERCRIT   above the critical temperature
+     */
+    int phaseState(bool checkState = false) const;
+
+    //! Return the value of the density at the liquid spinodal point (on the
+    //! liquid side) for the current temperature.
+    /*!
+     * @returns the density with units of kg m-3
+     */
+    virtual doublereal densSpinodalLiquid() const;
+
+    //! Return the value of the density at the gas spinodal point (on the gas
+    //! side) for the current temperature.
+    /*!
+     * @returns the density with units of kg m-3
+     */
+    virtual doublereal densSpinodalGas() const;
+
+public:
+    //! Calculate the saturation pressure at the current mixture content for the
+    //! given temperature
+    /*!
+     * @param TKelvin         (input) Temperature (Kelvin)
+     * @param molarVolGas     (return) Molar volume of the gas
+     * @param molarVolLiquid  (return) Molar volume of the liquid
+     * @returns the saturation pressure at the given temperature
+     */
+    doublereal calculatePsat(doublereal TKelvin, doublereal& molarVolGas,
+                             doublereal& molarVolLiquid);
+
+public:
+    //! Calculate the saturation pressure at the current mixture content for the
+    //! given temperature
+    /*!
+     * @param TKelvin   Temperature (Kelvin)
+     * @return          The saturation pressure at the given temperature
+     */
+    virtual doublereal satPressure(doublereal TKelvin);
+	virtual void setToEquilState(const doublereal* lambda_RT);

```

Comment:
Can you please add documentation for these two functions?

Reply:
Sorry, if these are pre-existing (I couldn't tell), then you don't need to worry about the documentation.

Reply:
Yes. These are pre-existing functions. I only moved these functions from Redlich-Kwong to MixtureFugacity class.